### ISSUE TITLE
Fix missing torsional parameters for O*-CT-CT-H*

### ISSUE THREAD

**Issue by davidlmobley** [open-forcefield-group/smarty#58]
In my continuing energy checking, @cbayly13 , I found that some torsional parameters from the parm@frosst frcmod file did not make it into the SMIRFF mockup and thereby not into the ffxml file. Particularly we seem to be missing `O*-CT-CT-H*` for OS, HC, H1, H2 at least... 

Note the frcmod file is missing a parameter for `O*-CT-CT-H2`, so it would have picked up the generic, yet another bug.

I'll at least need SMARTS to match these, though modifications to the ffxml files (the both the parm@frosst as it should be and parm@frosst as it was) via a PR are also welcome.

**Comment by cbayly13** [comment]
I just checked my smarff_mockup1.Frosst_AlkEthOH and I find (lines 51 and 52:
[#8X2]-[#6X4]-[#6X4]-[#1]   parTor ( (1, 0.00, 0.0, 3),
                                     (1, 0.25, 0.0, 1) ) parNum t0011 {H_-CT-CT-O_ from frcmod.Frosst_AlkEthOH, note parm@Frosst discrepancy with H2,H3-CT-CT-O_ }

For XML I suggest

``` XML
   <Proper smirks="[#1:1]-[#6X4:2]-[#6X4:3]-[OX2:4]" periodicity1="1" phase1="0.0" k1="0.25"/> <!-- HC,H1-CT-CT-OH,OS from frcmod.Frosst_AlkEthOH ; note corresponding H2 and H3 params missing so they presumably get generic parameters -->
```

**Comment by davidlmobley** [comment]
Thanks, resolved by #60 .
